Makefile

```diff
@@ -22,13 +22,14 @@ GRPID   := $(shell id -g)
 
 .PHONY: bin test-unit test-integration
 
+bin:
```

Comment:
nit: Prepend `.PHONY: bin` to it and move `.PHONY: test-unit test-integration` to L29 as other targets are in this style.

Reply:
Thanks, I have updated.